reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -632,4 +632,14 @@ class StaticHtmlReporter : Reporter() {
             }
         }
     }
+
+    private fun LI.errorDescription(error: ResolvableIssue) {
+        p {
+            var first = true
+            error.description.split("\n").forEach {
```

Comment:
How about using `lines()` instead of `split("\n")`?

Reply:
Done.